.github/workflows/ci.yaml

```diff
@@ -14,16 +14,20 @@ jobs:
       matrix:
         resolver:
           [
-            "lts-20.20",
+            "lts-21.15",
+            "lts-20.26",
             "lts-19.33",
             "lts-18.28",
-            "lts-16.31",
-            "lts-14.27",
```

Comment:
These don't support the new minimum bound on `template-haskell`.

Reply:
Is there a particular reason that you need to require `template-haskell >= 2.16`? The `Lift` class was introduced in `template-haskell-2.11.0.0` (GHC 8.0).

Reply:
Yes, it was due to this comment:

> Representation-polymorphic since template-haskell-2.16.0.0.

I wasn't sure exactly what this meant, so if you believe it's safe to lower the bound to `2.11` then I'll do so and add the earlier LTSs back to the test matrix.

Reply:
I've lowered it to `2.15` to account for `NonEmpty` only getting a `Lift` instance from that point.